src/style.css

```diff
@@ -1,3 +1,159 @@
-body {
+/* #region root */
+
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --gray-elements: #f3f3f3;
+  --background-main: #fff;
+  --starsCount: 4;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  color: var(--main-accent);
+}
+
+body,
+p,
+h2,
+h3 {
   margin: 0;
+  padding: 0;
+}
+
+/* #endregion */
+/* #region card main */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  height: 406px;
+  width: 198px;
+  border-radius: 5px;
+  background: var(--background-main);
+  border: 1px solid var(--gray-elements);
+}
+
+.card__container {
+  padding: 0 16px 16px;
+  width: fit-content;
+}
```

Comment:
please follow the BEM structure, check the checklist:
[BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.